plugin-server/tests/server.test.ts

```diff
@@ -66,8 +84,42 @@ describe('server', () => {
                     },
                 }
             )
+        })
+    })
+
+    test('starting and stopping node-schedule scheduled jobs', async () => {
+        expect(numberOfScheduledJobs()).toEqual(0)
+
+        pluginsServer = await createPluginServer()
+
+        expect(numberOfScheduledJobs()).toBeGreaterThan(1)
+
+        await pluginsServer.stop()
+        pluginsServer = null
+
+        expect(numberOfScheduledJobs()).toEqual(0)
+    })
+
+    describe('plugin-server capabilities', () => {
+        test('starts all main services by default', async () => {
+            pluginsServer = await createPluginServer()
+
+            expect(startPluginSchedules).toHaveBeenCalled()
+            expect(startJobQueueConsumer).toHaveBeenCalled()
+        })
+
+        test('disabling pluginScheduledTasks', async () => {
+            pluginsServer = await createPluginServer({}, { ingestion: true, pluginScheduledTasks: false, jobs: true })
+
+            expect(startPluginSchedules).not.toHaveBeenCalled()
+            expect(startJobQueueConsumer).toHaveBeenCalled()
+        })
+
+        test('disabling jobs', async () => {
```

Comment:
this is btw what led me to think about the jobs naming comment. this might sound like we're disabling jobs as in they can't even be created. In reality it's the consumer we're disabling.